### ISSUE TITLE
Duda seccion 3.2 Nueva compra 

### ISSUE THREAD

**Issue by Rocioldg** [IIC2413/Syllabus-2023-1#273]
Hola!

Cuando un cliente hace una compra, se debe actualizar la tabla stock?

No aparece explícitamente en el enunciado, por lo que no estamos seguros si es una funcionalidad extra o algo que se debe considerar.

**Comment by Paldana99** [comment]
Seria mas realista que lo hicieran efectivo

**Comment by Paldana99** [comment]
Ojo que en el punto del Admin se pide: 
> Este usuario puede generar una oferta para ciertos productos en una tienda, o
actualizar su stock.

**Comment by Larraguibel** [comment]
Exacto, el admin puede actualizar stock, pero no se nos pide que las compras del cliente afecten el stock. Independiente de si es mas realista es una funcionalidad extra? Independiente de si es más realista o no.

**Comment by Paldana99** [comment]
Si, no esta explicito en el enunciado por lo que seria opcional.
